apps/src/lab2/levelEditors/aichatSettings/EditAichatSettings.tsx

```diff
@@ -88,13 +89,31 @@ const EditAichatSettings: React.FunctionComponent<{
     [aichatSettings, setAichatSettings, initialCustomizations]
   );
 
+  const setModelSelectionValues = useCallback(
+    (additionalModelIds: Set<string>, selectedModelId: string) => {
+      const availableModelIds = Array.from(
```

Comment:
Nice :)

Reply:
Yeah, way better, thanks for pointing it out :-D